examples/plot_starting.py

```diff
@@ -0,0 +1,126 @@
+"""
+Starting an Active Learning experiment
+======================================
+
+Each active learning experiment starts with zero labeled samples. In that
+case, no supervised query strategy can be used. It is common to select the
+first batch of samples at random however it is proposed in
+`Diverse mini-batch Active Learning <https://arxiv.org/abs/1901.05954>`_
+to use a clustering approach for the first batch.
```

Comment:
Minor rephrasing
```suggestion
Active learning processes that start with zero labeled samples cannot
leverage supervised query strategy (building on top of trained model)
so that it is common to select the first batch of samples at random. 
However it was proposed in `Diverse mini-batch Active Learning <https://arxiv.org/abs/1901.05954>`_
to use a clustering approach for the first batch.
```